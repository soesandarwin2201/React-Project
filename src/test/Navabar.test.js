import { render } from '@testing-library/react';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navabar';

describe('navbar render correctly', () => {
  it('render correctly', () => {
    const { tree } = render(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
