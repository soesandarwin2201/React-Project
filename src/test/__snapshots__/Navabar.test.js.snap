// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar render correctly render correctly 1`] = `undefined`;
